near/contracts/token-bridge/src/lib.rs

```diff
@@ -1652,32 +1654,30 @@ impl TokenBridge {
         }
     }
 
-    #[init(ignore_state)]
-    #[payable]
-    pub fn migrate() -> Self {
-        if env::attached_deposit() != 1 {
-            env::panic_str("Need money");
-        }
-        let old_state: OldPortal = env::state_read().expect("failed");
-        if env::signer_account_pk() != old_state.owner_pk {
-            env::panic_str("CannotCallMigrate");
-        }
-        env::log_str(&format!("token-bridge/{}#{}: migrate", file!(), line!(),));
-        Self {
-            booted:               old_state.booted,
-            core:                 old_state.core,
-            gov_idx:              0,
-            dups:                 LookupMap::new(b"d".to_vec()),
-            owner_pk:             old_state.owner_pk,
-            emitter_registration: old_state.emitter_registration,
-            last_asset:           old_state.last_asset,
-            upgrade_hash:         old_state.upgrade_hash,
-            tokens:               old_state.tokens,
-            key_map:              old_state.key_map,
-            hash_map:             old_state.hash_map,
-            bank:                 old_state.bank,
-        }
-    }
+//    #[init(ignore_state)]
```

Comment:
Should this just go away?

Reply:
yes, in the future we might want to do a migration so I wanted to keep it around BUT I didn't want it in the contract in case there was a unknown security issue with this... pure paranoia which is why I commented it out